packages/input/src/ux-input.html

```diff
@@ -22,5 +22,5 @@
 
   <slot name="trailing-icon"></slot>
 
-  <div class="ux-input__border-bottom"></div>
+  <div class="ux-input__border-bottom" show.bind="variant === 'filled' && !disabled"></div>
```

Comment:
shouldn't there be a class binding for active as well? I remember there used to be 2 divs in old commits. So it would look like this:
```html
<div class="ux-input__border-bottom${variant === 'outline' ? 'active' : ''}" show.bind="variant === 'filled' || variant === 'outline' && !disabled"></div>
```

Reply:
@bigopon

There was 2 divs to display the 2 lines. Now they are replaced by 2 pseudo elements (:before and :after) and the active thing is in fact handled in CSS with the `.ux-input__inner-input:focus` selector (see below)

```css
.ux-input__inner-input:focus~.ux-input__border-bottom:after {
  transform: scaleX(1);
}
```

Note: there is an open discussion about using `:focus` or BEM style `...--focused` CSS. But in any case, this can be handled without a specific class on the border element.

Reply:
@ben-girardet thanks, I forgot this. Probably that's better